app/javascript/mastodon/features/ui/components/media_modal.js

```diff
@@ -73,7 +72,7 @@ class MediaModal extends ImmutablePureComponent {
     }
 
     if (attachment.get('type') === 'image') {
-      content = <ImageLoader src={url} imgProps={{ style: { display: 'block' } }} />;
+      content = <img src={url} />;
```

Comment:
The purpose of both [`alt=''`](https://www.w3.org/WAI/tutorials/images/decorative/) and [`role='presentation'`](http://asurkov.blogspot.com/2012/02/aria-hidden-and-rolepresentation.html) (as I understand it) is to just hide the `img` from ATs. This particular `img` is the modal dialog for a user-uploaded image, and to me it really ought to have an `alt` supplied by the user, but unfortunately Mastodon doesn't provide any way to do this currently.

In any case the existing behavior of `ImageLoader` is to have neither `alt` nor `role` so the behavior is unchanged.